.github/dependabot.yml

```diff
@@ -6,4 +6,12 @@ updates:
       interval: daily
       time: "09:00"
       timezone: America/New_York
+      ignored_updates:
+      - match:
+          dependency_name: "core-js"
+          dependency_name: "@babel*"
```

Comment:
I made it so that these packages are automatically determined via the configuration: 
https://github.com/redwoodjs/redwood/blob/4597b5d4534900889aa2ddf5b4cb93e4fea0721d/packages/core/config/babel-preset.js#L32

We can remove these now :)